firmware/boron-ins-fsm/src/imDoorSensor.cpp

```diff
@@ -69,6 +70,11 @@ doorData checkIM() {
         // Checks if the 0th bit (counting from 0) of doorStatus is 1
         doorTamperedFlag = (currentDoorData.doorStatus & 0b0001) != 0;
 
+        // Set timer starting when door closes
+        if ((previousDoorData.doorStatus & 0b0010) != 0 && (currentDoorData.doorStatus & 0b0010) == 0) {
+            timeSinceDoorClosed = millis();
```

Comment:
Yup, using `millis()` is exactly how I would do this too. It sets the value to **the time at which the door closed**.

However, when I read the variable name `timeSinceDoorClosed` I would have expected this to set it to 0. The door just closed; therefore, **the time since the door closed** is 0 milliseconds.

So I think this variable could have a clearer name to reduce the likelihood of confusion.